lib/nostrum/consumer.ex

```diff
@@ -131,9 +131,8 @@ defmodule Nostrum.Consumer do
            {guild_id :: integer, old_role :: Nostrum.Struct.Guild.Role.t() | nil,
             new_role :: Nostrum.Struct.Guild.Role.t()}, WSState.t()}
   @type message_create :: {:MESSAGE_CREATE, {message :: Nostrum.Struct.Message.t()}, WSState.t()}
-  @type message_delete :: {:MESSAGE_DELETE, {message :: MessageDelete.t()}, WSState.t()}
-  @type message_delete_bulk ::
-          {:MESSAGE_DELETE_BULK, {updated_messages :: [Nostrum.Struct.Message.t()]}, WSState.t()}
+  @type message_delete :: {:MESSAGE_DELETE, {MessageDelete.t()}, WSState.t()}
```

Comment:
I didn't think of this before, but if we're including everything in structs we should be able to drop the tuple. This would be a breaking change, but I think we should go ahead with it regardless. :^)

Reply:
Will do. I'll create an issue for making sure the tuple is removed in future structs.

Reply:
Just remembered something. If we are introducing breaking changes, then we can take this opportunity to make a less verbose message passing format. Here is what we have right now:
```elixir
{event_atom, {payload}, WSState.t()}
# Example:
{:MESSAGE_DELETE, {%MessageDelete{...}}, %WSState{...}}
```
The struct type and `event_atom` tack on extra information. In addition to removing the tuple, we can reasonably reduce the event format to the following:
```elixir
{payload, WSState.t()}
# Example:
{%MessageDelete{...}, %WSState{...}}
```
This is an easy change that can fit within the scope of this PR. Let me know what you think.

Reply:
I think thematically this is a good change. My worries are that it makes things a little less explicit. As this `handle_event` is the defacto way that users interact with events from discord I think it's nice to be able to see that atom that refers directly to the event as opposed to a struct type. This is however just my opinion - if we're moving to structs everywhere then this might be the play to make. Let me know what you think and we can see if any others want to chime in.

As for fitting it into this pr, I think we should hold off for now. I'm okay with breaking changes for now, but I think we should just take care of all of these in one fell swoop after all the event types are sorted out.

Reply:
Then we'll hold off on this change for now. I'll move on with removing the tuple.

Reply:
Resolved in 2d487ca.